src/domain/pricing/details/sections/prices-details/edit-prices-overrides/index.tsx

```diff
@@ -80,6 +80,7 @@ const EditPricesOverridesModal = ({
                     key={variant.id}
                     onClick={getOnClick(variant)}
                     variant={variant}
+                    prices={variant.prices.filter((pr) => pr.price_list_id)}
```

Comment:
question: In case the variant is part of more than one price list, would we not end up showing prices for all of those in the modal with the current filtering?
```suggestion
                    prices={variant.prices.filter((pr) => pr.price_list_id === priceListId )}
```

Reply:
correct me if I'm wrong @olivermrbl but for me it seems like that never happens. Even when a variant is part of more than one price list, what we end up with is default prices + prices defined in that particular price list. Does it behave in a different way for you?

Reply:
Got it! My bad :)